Makefile

```diff
@@ -0,0 +1,64 @@
+.PHONY: default
+default: help
+
+.PHONY: help
+help:
+	@echo "make help              Show this help message"
+	@echo "make dev               Run the pattern library application (TODO)"
+	@echo "make build             Build the package"
+	@echo "make lint              Run the code linter(s) and print any warnings"
+	@echo "make checkformatting   Check code formatting"
+	@echo "make format            Automatically format code"
+	@echo "make test              Run the unit tests once"
+	@echo "make sure              Make sure that the formatter, linter, tests, etc all pass"
+	@echo "make clean             Delete development artefacts (cached files, "
+	@echo "                       dependencies, etc)"
+
+.PHONY: dev
+dev: node_modules/.uptodate
+	@echo 'TODO: run local dev server with pattern library'
+
+.PHONY: test
+test: node_modules/.uptodate
+ifdef ARGS
+	yarn test $(ARGS)
+else
+	yarn test
+endif
+
+.PHONY: lint
+lint: node_modules/.uptodate
+	yarn run lint
+	yarn run typecheck
+
+.PHONY: docs
+docs: python
+	tox -e py36-docs
+
+.PHONY: checkdocs
+checkdocs: python
+	tox -e py36-checkdocs
+
+.PHONY: clean
+clean:
```

Comment:
Can you also make this remove the /lib folder?

Reply:
I think this generally sounds like a sound idea, but since I'm unfamiliar with the build process that generates `lib` in the first place (and thus unable to meaningfully test that I'm not breaking things), would you mind taking this as a follow-up?

Reply:
Yep!